dotcom-rendering/src/web/components/SignInGate/displayRule.ts

```diff
@@ -102,6 +120,41 @@ export const hasRequiredConsents = (): Promise<boolean> => {
 	});
 };
 
+/**
+ * Occasionally we may want to track long-running behavior of browsers in a certain test,
+ * without adding new browsers to the test bucket. We cannot rely on the GU_mvt_id bucketing alone to do this.
+ * You can use this method to show the gate based on local storage participation key ("gu.ab.participations") instead.
+ *
+ * After the cut off date users without the local storage participation key won't see the gate at all.
+ *
+ * @param cutOff - UTC timestamp
+ * @param currentTest - test the
+ * @returns Promise<boolean>
+ */
+export const useParticipations = (
+	cutOff: number,
+	currentTest: CurrentSignInGateABTest,
+): Promise<boolean> => {
+	const now = Date.now(); // UTC
+	const { id, variant } = currentTest;
+
+	// BEFORE cutoff date all users will be bucketed and potentially shown the gate if they satisfy all subsequent conditions
+	if (now < cutOff) {
+		// sets participation under local storage key: 'gu.ab.participations'
+		setParticipationsInLocalStorage({
+			[id]: { variant: variant },
+		});
+		return Promise.resolve(true); //  we only want to bucket until the cutoff
+	}
+
+	// AFTER cutoff date, browsers without local storage test participation will not be shown the gate
+	const participations = getParticipationsFromLocalStorage();
+	// if the test participation exists, show the test gate
+	return !!participations[id] && participations[id]?.variant === variant
+		? Promise.resolve(true)
+		: Promise.resolve(false);
```

Comment:
The conditional chaining helps you avoid the check for the existence of `participations[id]`.

```suggestion
	return participations[id]?.variant === variant
		? Promise.resolve(true)
		: Promise.resolve(false);
```